src/components/ConnectWidgets.tsx

```diff
@@ -15,7 +15,8 @@ export type ConnectWidgetProps = SsoUrlProps &
   OAuthProps &
   LoadUrlInBrowserProps &
   ConnectPostMessageCallbackProps<string> &
-  ConnectWidgetConfigurationProps
+  ConnectWidgetConfigurationProps &
+  JSX.IntrinsicAttributes
```

Comment:
Needed after migrating from React v17 to v18 (SDK uses 17, example app uses 18). It's possible we'll be able to revert this change after upgrading the SDK.